aptos-move/e2e-move-tests/src/tests/keyless_feature_gating.rs

```diff
@@ -46,6 +47,12 @@ fn init_feature_gating(
 
     // initialize JWKs
     let core_resources = run_jwk_and_config_script(&mut h);
+    // initialize default VK
+    run_upgrade_vk_script(
+        &mut h,
+        core_resources.clone(),
+        Groth16VerificationKey::from(DEVNET_VERIFICATION_KEY.clone()),
```

Comment:
`DEVNET_VERIFICATION_KEY` is the hardcoded VK, no?

Reply:
Oh, I should rename this variable now, since it's more of a `TESTING_VERIFICATION_KEY`.

It is no longer hardcoded / used as a the default devnet VK.

Reply:
Fixed.

Reply:
Actually, this surfaced an incomplete feature in the pepper service. I tagged you; please take a look!